### PR TITLE
feat(docs): deduplication feature

### PR DESCRIPTION
**Pull request by ideoma**
Deduplication concept and SQL reference for PR

https://github.com/questdb/questdb/pull/3566

### CODE REVIEW

**Review by marregui — APPROVED** [review]
LGTM I am unable to find anything wrong with it. It reads very clear.
If anything, 

```
Deduplication can be enabled or disabled at any time for individual tables using the following statements:

[CREATE TABLE](http://localhost:3000/docs/reference/sql/create-table/#deduplication) statement
[ALTER TABLE ENABLE DEDUP](http://localhost:3000/docs/reference/sql/alter-table-enable-deduplication) statement
[ALTER TABLE DISABLE DEDUP](http://localhost:3000/docs/reference/sql/alter-table-disable-deduplication) statement
```
you could remove the trailing `statement` for each item of the list, as the hearing already says that the items are statements.

**Review by argshook — APPROVED** [review]

**Comment by bluestreak01** [comment]
@ideoma there is minor consistency issue:

**Review by goodroot — DISMISSED** [review]
LGTM!

File: docs/reference/sql/create-table.md [hunk 1]
```diff
@@ -217,6 +217,24 @@ and modification through multiple interfaces:
...
+
+However, there are a few limitations to keep in mind:
+
+- The designated timestamp column must be included in the list of columns
+- Columns of [STRING and BINARY](/docs/reference/sql/datatypes) types cannot be used in `UPSERT KEYS` list
+
+After table creation the deduplication configuration can be changed at any time using `ALTER` table:
+
+- Enable deduplication and change `UPSERT KEYS` with [`ALTER TABLE ENABLE`](/sql/alter-table-enable-deduplication/)
+- Disable deduplication with using [`ALTER TABLE DISABLE`](/sql/alter-table-disable-deduplication/)
```
Comment (goodroot) [inline docs/reference/sql/create-table.md:237]:
```suggestion
- Enable deduplication and change `UPSERT KEYS` with [`ALTER TABLE ENABLE`](/docs/sql/alter-table-enable-deduplication/)
- Disable deduplication with using [`ALTER TABLE DISABLE`](/docs/sql/alter-table-disable-deduplication/)
```

File: docs/reference/sql/create-table.md [hunk 2]
```diff
@@ -217,6 +217,24 @@ and modification through multiple interfaces:
...
+
+However, there are a few limitations to keep in mind:
+
+- The designated timestamp column must be included in the list of columns
+- Columns of [STRING and BINARY](/docs/reference/sql/datatypes) types cannot be used in `UPSERT KEYS` list
+
+After table creation the deduplication configuration can be changed at any time using `ALTER` table:
+
+- Enable deduplication and change `UPSERT KEYS` with [`ALTER TABLE ENABLE`](/docs/sql/alter-table-enable-deduplication/)
+- Disable deduplication with using [`ALTER TABLE DISABLE`](/docs/sql/alter-table-disable-deduplication/)
```
Comment (goodroot) [inline docs/reference/sql/create-table.md:237]:
```suggestion
- Enable deduplication and change `UPSERT KEYS` with [`ALTER TABLE ENABLE`](/docs/reference/sql/alter-table-enable-deduplication/)
- Disable deduplication with using [`ALTER TABLE DISABLE`](/docs/reference/sql/alter-table-disable-deduplication/)
```

**Review by goodroot — APPROVED** [review]
LGTM

**Pull request merged**